src/Json.Schema.ToDotNet/TypeGenerator.cs

```diff
@@ -73,12 +73,12 @@ public string Generate(string namespaceName, string typeName, string copyrightNo
 
         protected void AddUsing(string namespaceName)
         {
-            if (Usings == null)
+            Usings = Usings ?? new List<string>();
+
+            if (!Usings.Contains(namespaceName))
```

Comment:
>Contains [](start = 24, length = 8)

Avoid duplicates in this list. This has never been a problem; there's probably code somewhere else that de-dupes it, but I can't be troubled to look for it now.